tools/rosbag/src/player.cpp

```diff
@@ -278,6 +281,25 @@ void Player::printTime()
     }
 }
 
+bool Player::pauseCallback(std_srvs::SetBool::Request &req, std_srvs::SetBool::Response &res)
+{
+  pause_change_requested_ = (req.data != paused_);
+  requested_pause_state_ = req.data;
+
+  res.success = pause_change_requested_;
+
+  if (res.success)
+  {
+    res.message = (requested_pause_state_ ? "Pause" : "Resume") + std::string(" requested.");
```

Comment:
Wondering if this should be past tense? Eg, `Bag playback is now paused/resumed.`